public/app/plugins/datasource/loki/components/AnnotationsQueryEditor.tsx

```diff
@@ -22,7 +22,7 @@ export const LokiAnnotationsQueryEditor = memo(function LokiAnnotationQueryEdito
     to: Date.now(),
   };
 
-  const { isSyntaxReady, setActiveOption, refreshLabels, syntax, logLabelOptions } = useLokiSyntaxAndLabels(
+  const { setActiveOption, refreshLabels, logLabelOptions, labelsLoaded } = useLokiLabels(
     datasource.languageProvider,
```

Comment:
labelsLoaded is replacing the isSyntaxReady to determine the text of cascader button